packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/glue/start-job-run.ts

```diff
@@ -64,8 +64,19 @@ export interface GlueStartJobRunProps extends sfn.TaskStateBaseProps {
 export interface WorkerConfigurationProperty {
   /**
    * The type of predefined worker that is allocated when a job runs.
+   *
+   * @default - must choose one of `workerType` or `workerTypeV2`
+   * @deprecated Use `workerTypeV2` for more flexibility in defining worker types.
    */
-  readonly workerType: WorkerType;
+  readonly workerType?: WorkerType;
+
+  /**
+   * The type of predefined worker that is allocated when a job runs. Can be one of the
+   * predefined values or dynamic values using `WorkerTypeV2.of(...)`.
+   *
+   * @default - must choose one of `workerType` or `workerTypeV2`
+   */
+  readonly workerTypeV2?: WorkerTypeV2;
```

Comment:
I wonder do we have a naming pattern for this kind of change? e.g. `workerTypeV2` or `workerTypeEnum`